.github/workflows/go.yml

```diff
@@ -44,6 +44,3 @@ jobs:
           go get -u -v github.com/anz-bank/sysl/cmd/sysl
           go get -u -v github.com/arr-ai/arrai/cmd/arrai
           go get -u -v golang.org/x/tools/cmd/goimports
-
-      - name: Compare Sysl and arr.ai transforms output
```

Comment:
Please also remove the `arrai-nodiff` rule in `make`.

Reply:
ok